app/domains/fluggastrechte/services/pdf/sections/reason/addNewPageInCaseMissingVerticalSpace.ts

```diff
@@ -1,13 +1,31 @@
 import type PDFDocument from "pdfkit";
 import { PDF_HEIGHT_SEIZE } from "~/services/pdf/createPdfKitDocument";
 
-const MAX_VERTICAL_SPACE = PDF_HEIGHT_SEIZE - 80;
+const MAX_VERTICAL_SPACE = PDF_HEIGHT_SEIZE - 70;
+//doc.moveDown(1) moves down 20 dpi
+const DEFAULT_MOVE_DOWN = 20;
+
+type AddNewPageInCaseMissingVerticalSpaceParams = {
```

Comment:
```suggestion
type PageBreakParams = {
```
or is it too simple?